### ISSUE TITLE
Equitable  access to and availability of platforms and tools used for collaboration in online communities

### ISSUE THREAD

**Issue by serahrono** [carpentries/conversations#26]
There have been pockets of discussion within The Carpentries community around this subject in the last year or so, and these four events stand out as our 'why' for putting out a more formal request for community input around this subject at  this time:
+ While writing a communications strategy for The Carpentries in Q1 2019, we heard from community members that certain tools and platforms that The Carpentries uses are inaccessible to community members in certain countries and regions around the world. 
+ In December 2019, as our community chatted about The Carpentries teaching of Git and use of GitHub ([see blog post](https://carpentries.org/blog/2020/05/conversations-teaching-git-github/)), this issue was also raised in discussions. 
+ In June 2020, while preparing for 2020.carpentrycon.org, CarpentryCon Task Force checked in with some community members to determine what alternative platforms could be used during sessions in order to make the virtual conference a more inclusive experience.
+ In August 2020, Steve van Tuyl posed a question in [The Carpentries' Slack](https://swc-slack-invite.herokuapp.com/) '_about where to find out what countries restrict access to what software applications/web applications_' and with an aim to use this to guide the organising of a virtual hack week with international communities participating

We would like to hear from you: 

**what tools and platforms used by The Carpentries are inaccessible or unavailable to people in certain regions around the world?** 

Please share as much detail as you can, with links where possible, and remember to mention affected countries / regions in your response.

We hope to gather input over the next few weeks, as we work to organise for community discussions to deliberate, and determine a way forward.

**Comment by k8hertweck** [comment]
Sean Davis shared the following link in an email to this message to the discuss listserv:

> This might be helpful objective information as well:
> 
> https://docs.github.com/en/github/site-policy/github-and-trade-controls

**Comment by selgebali** [comment]
- Tools for virtual conferences

 https://docs.google.com/document/d/1LLLniPkf48CCZyG_BNy1ylF2wXNlztqNEOnzNuMQmJc/edit 

- Creating an Online Community, Class or Conference - Quick Tech Guide https://docs.google.com/document/d/1ux3lTnUTpzZRuvxE3rAsSQ4Ihub96S8_OYECNh8wv-A/edit#

- The Ultimate Webinar Livestreaming List https://docs.google.com/spreadsheets/d/1lvDY4yYBEeAABQlIpHrjxXxBJ7ZtjQGj7ILp6p2vqdY/edit?ts=5eb19d9b#gid=0 

-This one is less related but interesting nevertheless:  excellent thread on accessibility within content management systems (like Blackboard etc)
https://twitter.com/AimiHamraie/status/1237057301204197377

**Comment by LJWilliams** [comment]
Surprisingly enough there can be bandwidth issues in Northern communities in Canada too.  In major centres (Toronto, Montreal, Ottawa, Vancouver, Calgary, etc.) one can opt to pay for higher bandwidth, but many areas still do not have cell service, let alone the internet bandwidth necessary to run software such as Zoom.  

On another note, has anyone looked at [Jitsi](https://jitsi.org/) as an alternative to Zoom? Free, open source, etc. etc. I'm not sure it solves the bandwidth issue though.

**Comment by vantuyls** [comment]
It may also be useful to include restrictions that federal and state employees (in the US and elsewhere) face when attending workshops. My experience has been that government employees tend to have their machines "locked down", causing disruption in their learning. I'm not exactly sure how to address this, since these regulations could vary from state to state or agency to agency.

**Comment by zpainter** [comment]
I have wondered, other than the technical debt of having to move platforms entirely, given that most of our teaching is designed to be using FOSS tools, why we continue to use GitHub instead of GitLab? I realize the irony of posting this on GitHub, of course. But if we can't use GitHub around the world, and we could use GitLab, doesn't it make more sense to just use that? We probably bring a fair amount of weight to user communities with our choice of platforms, and it would be encouraging for us in some ways to have people use GitLab.

As for locked down machines, the Data Carpentry Genomics lesson might offer a path for that in some ways. Building a Docker container or something similar and just hosting everything in it via a browser might get most of the functionality we need to teach a workshop over without needing much in the way of admin rights. I have toyed a bit with this myself, but haven't fully fleshed out what the end product would look like.

**Comment by sdopoku** [comment]
This is a great discussion and I'm very grateful for all the resources that have been shared so far; many that I will explore for my work going forward.

I don't have particular tools or technologies to propose (aside what has been listed above) but from my work in resource-constrained environments, what has been helpful is involving the very stakeholders in the inaccessible contexts we hope to reach in these conversations and decisions at the start. Most of the time, we have found out that they already have a workaround or something close which we can collaborate with them to optimize further. Some approaches have been baking in in-depth foundational skills on tools, allocating funds for faster internet data packages, or plainly switching to something totally new to us. 

For those that do not have a workaround, it becomes a decision on the pros and cons of each tool for their context. For collaborative tools (for instance GitHub vs GitLab), it may be beyond a one-size-fits-all solution. What may be accessible through GitLab for a specific group, may not be the case for another and vice versa. The challenge here for The Carpentries is being able to manage the community and the source of truth if multiple platforms are being used. It may however be worth trying to understand these frictions and what combination increases accessibility given the community's capacity to manage it.

In summary, a great start could be:
1. Listing the locations or contexts our target stakeholders are located in
2. Working with them to identify the pros and cons of current tools or resources they use (and not use) when collaborating
3. Identifying those tools that are universally accessible, and making available a stack of context-specific tools and resources that achieve the collaborative goals for each stakeholder.

I'm happy to help with this once a decision on made on how to go forward.

PS: I've found the [Alliance for Affordable Internet](https://a4ai.org/) a great resource to keep track of the issues of internet access. Their [research and publications](https://a4ai.org/research/) especially **The Affordability Report** and **Mobile Broadband Pricing Data** are worth checking out.

**Comment by zkamvar** [comment]
I know I'm late to the party, but **I think @sdopoku really highlighted the salient points**. There are no universal tools that will work for everyone and global politics ensures that fact. *I don't have any answers*, but I think the axes of accessibility we are dealing with are: 

1. regionally accessible via government control
2. accessible via bandwidth restrictions
3. accessible via language fluency
4. accessible to people who are neurodivergent, blind, deaf, and hard of hearing (e.g. [Google Meet has a better captioning system than Zoom](https://twitter.com/IndigenousAI/status/1316533117130141697?s=20))

IMO, when investigating The Carpentries infrastructure, it's important to see what platforms actually have stated policies about their limitations for use around these axes. If they do not have explicit policies, that indicates that they have no explicit strategy for that particular aspect of accessibility.

(please let me know if I missed the point, entirely, though)

**Comment by ElnazAmanzadeh** [comment]
Hi all,
This is Elnaz. I am from Iran. I have run the first DC Genomics workshop in Iran. We had an issue using AWS. Appearently, IP of Iran is blocked and service isnt available for us.
I would prefer to prvide the possibility for my students to teach them cloud computing. I was wondering if anyone else had the similar issues and if had found a solution, or tried any other services, please let me know.
Thanks
Elnaz

**Comment by ErinBecker** [comment]
> Hi all,
> This is Elnaz. I am from Iran. I have run the first DC Genomics workshop in Iran. We had an issue using AWS. Appearently, IP of Iran is blocked and service isnt available for us.
> I would prefer to prvide the possibility for my students to teach them cloud computing. I was wondering if anyone else had the similar issues and if had found a solution, or tried any other services, please let me know.
> Thanks
> Elnaz

Thanks for letting us know about this @ElnazAmanzadeh. Our team discussed this issue and have a couple of possible solutions we would like to suggest. 

1) The first solution is to use the detailed installation instructions listed in [option B: using the lessons on your local machine](https://datacarpentry.org/genomics-workshop/setup#option-b-using-the-lessons-on-your-local-machine). If you have access to a compute cluster at your institution, and are able to do the install there, this option may work for your learners.

2) An alternative is to try using a Docker image. @JasonJWilliamsNY and @naupaka were working on a [Docker image for the DC Genomics lessons](https://github.com/JasonJWilliamsNY/dc_genomics_docker) two years ago, but I don't know if that work has progressed. 

Thank you again for letting us know about the problem. If you do explore either of these options, or come up with a different solution, please let us know! You can stay in touch through this issue or by emailing team@carpentries.org.

**Comment by naupaka** [comment]
I don't think there has been much progress on the Docker image, unfortunately, although it is still on our list of things to do. @JasonJWilliamsNY can correct me if I'm wrong.

**Comment by JasonJWilliamsNY** [comment]
I believe there is a functional docker  image. @ErinBecker do you know what is running on the AWS machines now? It should not take much effort to update if needed. Would be happy to help.

**Comment by ErinBecker** [comment]
That sounds great @JasonJWilliamsNY. Could we find a half hour to meet in January to go through what needs to be done and what info you need from me? [Here's my calendar](https://calendly.com/ebecker-1/30min?back=1&month=2021-01), but happy to schedule on yours if easier for you.
